tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -399,9 +399,13 @@ const reducer = createReducer(
         ? !partialSettings.timeSeriesPromotionDismissed
         : state.promoteTimeSeries;
 
+    const isSettingsPaneOpen =
+      partialSettings.timeSeriesSettingsPaneOpened ?? state.isSettingsPaneOpen;
```

Comment:
somehow I did not see `isSettingsPaneOpen: true,` around line 265 in this pr. probably need rebase before merge